server/config.ts

```diff
@@ -72,6 +72,24 @@ export default {
       agent: new AgentConfig(Number(get('TOKEN_VERIFICATION_API_TIMEOUT_RESPONSE', 5000))),
       enabled: get('TOKEN_VERIFICATION_ENABLED', 'false') === 'true',
     },
+    suicideRisk: {
+      url: get('SUICIDE_RISK_API_URL', 'http://localhost:9091/suicide-risk-api', requiredInProduction),
+      healthPath: '/health/ping',
+      timeout: {
+        response: Number(get('SUICIDE_RISK_API_TIMEOUT_RESPONSE', 5000)),
+        deadline: Number(get('SUICIDE_RISK_API_TIMEOUT_DEADLINE', 5000)),
+      },
+      agent: new AgentConfig(Number(get('SUICIDE_RISK_API_TIMEOUT_RESPONSE', 5000))),
+    },
+    ndeliusIntegration: {
+      url: get('NDELIUS_INTEGRATION_URL', 'http://localhost:9091', requiredInProduction),
```

Comment:
Might be worth adding a path for this, same as you've done for the suicide risk api
```suggestion
      url: get('NDELIUS_INTEGRATION_URL', 'http://localhost:9091/ndelius', requiredInProduction),
```

It will probably help to differentiate between the WireMock mappings - especially given you'll be integrating with more APIs in this service than you did in breach notice.